src/rpc/mining.cpp

```diff
@@ -1005,6 +1006,90 @@ static UniValue getblocktemplate(const Config &config,
     return result;
 }
 
+static UniValue getunsolvedblock(const Config &config,
+                                 const JSONRPCRequest &request) {
+    RPCHelpMan{
+        "getunsolvedblock",
+        "Get a block candidate header with coinbase sending to specified "
+        "address.",
```

Comment:
Change to something like:
```suggestion
        "Get a serialized block candidate with coinbase sending to the specified address.",
```